imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -742,7 +742,7 @@ func Test_fdPrestatDirName(t *testing.T) {
 
 	requireErrno(t, ErrnoSuccess, mod, fdPrestatDirNameName, uint64(fd), uint64(path), uint64(pathLen))
 	require.Equal(t, `
-==> wasi_snapshot_preview1.fd_prestat_dir_name(fd=3,path=1,path_len=0)
+==> wasi_snapshot_preview1.fd_prestat_dir_name(fd=3,result.path=1,result.path_len=0)
```

Comment:
this was unrelated, but I noticed it. fd_prestat_dir_name "path" is an out parameter, as it is written by the function fd_prestat_dir_name, not its caller.